src/javascript/app/pages/user/account/settings/personal_details.js

```diff
@@ -129,6 +129,20 @@ const PersonalDetails = (() => {
         }
     };
 
+    const selectContactCSMessage = (immutable_fields) => {
+        const has_name = /name/.test(immutable_fields.toString());
+        const has_tax_info = /tax/.test(immutable_fields.toString());
+        if (has_name && has_tax_info) {
+            $('.rowCustomerSupport').not('.full-cs-msg').setVisibility(0);
```

Comment:
A question: why don't we hide all the messages by default and then make one visible in this function? Currently, if this logic somehow fails it's going to show all of the messages at the same time 😄

Reply:
all of these  are invisible by default, she put class invisible on each of these elements, @maryia-deriv should these be `setVisibility(1)`?

Reply:
True, they are invisible, but here it just makes them invisible again 😕

Reply:
**The current flow is the following:**
1) First, all elements for authenticated account only (marked with the `.RealAccAuth` className) are made invisible including unnecessary Contact CS strings because if there are any other elements marked with the `.RealAccAuth` className, they also must be shown; _(line 183 in personal_details.js)_
2) Then, I hide all unnecessary Contact CS strings (among those marked with the `.rowCustomerSupport` className) while leaving the relevant Contact CS string only. _(line 184 in personal_details.js)_
**The reason for doing that:**
I suppose that the author of the initial code designed the `.RealAccAuth` className to mark any elements that must be visible for authenticated accounts only.
However, for some reason, the .RealAccAuth className is now used for the Contact CS string only. If you think that there will be no additional elements in the future that must be also visible only for authenticated accounts, I can remove the `.RealAccAuth` className or the `.rowCustomerSupport` className because at the moment both of them are used by Contact CS string only. And in such a case, knowing that the Contact CS string is the only element that must visible for authenticated accounts only, I will be able to first hide unnecessary Contact CS strings, and then show the relevant one. What do you suggest?

Reply:
Why can't we just remove the line 183? I see that `.RealAccAuth` only refers to these paragraphs and nothing else, so it's just an extra manipulation with the DOM by making them invisible by default, showing them all, and then removing each one except the correct element.

Reply:
@akmal-binary Thank you. I have removed a redundant .RealAccAuth className according to your suggestion)